packages/bpk-component-navigation-bar/src/BpkNavigationBar.module.scss

```diff
@@ -41,40 +48,45 @@
   }
 
   &__title {
+    grid-column: 2;
+    text-align: center;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+
     @include utils.bpk-themeable-property(
       color,
       --bpk-navigation-bar-title-color,
       tokens.$bpk-text-primary-day
     );
 
+    // ensure we still apply ellipsis overflow when a custom title element is specified e.g. h2
```

Comment:
BottomSheet is an example where we want an h2 element but it needs to be treated as inline to clamp the text